webpack.config.js

```diff
@@ -404,6 +445,74 @@ function* webpackConfig( env, argv ) {
 			} ),
 		],
 	};
+
+	// Build the Gutenberg entry points.
+	yield {
+		entry: gutenbergEntryPoints,
+		externals: gutenbergExternals,
+		output: {
+			filename:
+				mode === 'production' ? '[name]-[contenthash].js' : '[name].js',
+			path: path.join( __dirname, 'dist/assets/js' ),
+			chunkFilename:
+				mode === 'production' ? '[name]-[chunkhash].js' : '[name].js',
+			publicPath: '',
+			// If multiple webpack runtimes (from different compilations) are used on the
+			// same webpage, there is a risk of conflicts of on-demand chunks in the global
+			// namespace.
+			// See: https://webpack.js.org/configuration/output/#outputjsonpfunction.
+			jsonpFunction: '__googlesitekit_block_editor_webpackJsonp',
+		},
+		performance: {
+			maxEntrypointSize: 175000,
+		},
+		module: {
+			rules: createRules( mode ),
+		},
+		plugins: [
+			new WebpackBar( {
+				name: 'Gutenberg Entry Points',
+				color: '#ffc0cb',
+			} ),
+			new CircularDependencyPlugin( {
+				exclude: /node_modules/,
+				failOnError: true,
+				allowAsyncCycles: false,
+				cwd: process.cwd(),
+			} ),
+			new ManifestPlugin( {
+				...manifestArgs,
+				filter( file ) {
+					return ( file.name || '' ).match( /\.js$/ );
+				},
+			} ),
+			new ESLintPlugin( {
+				emitError: true,
+				emitWarning: true,
+				failOnError: true,
+			} ),
+		],
+		optimization: {
+			minimizer: [
+				new TerserPlugin( {
+					parallel: true,
+					sourceMap: mode !== 'production',
+					cache: true,
+					terserOptions: {
+						// We preserve function names that start with capital letters as
+						// they're _likely_ component names, and these are useful to have
+						// in tracebacks and error messages.
+						keep_fnames: /__|_x|_n|_nx|sprintf|^[A-Z].+$/,
+						output: {
+							comments: /translators:/i,
+						},
+					},
+					extractComments: false,
+				} ),
```

Comment:
Looks like this plugin is identical to the one we already use in the main config. Let's assign TerserPlugin options to a variable and use it for both instances. Or maybe create a function that will return a new instance of the TerserPlugin plugin. Up to you.